### PR TITLE
Add defaults for params and constants in pyscreeze

### PR DESCRIPTION
**Pull request by Avasam**
1. Ran https://github.com/JelleZijlstra/stubdefaulter
2. Manually added default kwarg parameters
3. Manually added defaults and Final for constants

### CODE REVIEW

**Comment by JelleZijlstra** [comment]
> Manually added default kwarg parameters

Hm is there something missing in stubdefaulter around this? If so please let me or Alex know so we can fix it.

**Comment by Avasam** [comment]
> > Manually added default kwarg parameters
> 
> Hm is there something missing in stubdefaulter around this? If so please let me or Alex know so we can fix it.

Nah, it's **kwargs passed to another function. We expand the kwargs in the stubs. Not anything stubdefaulter could know about (nor does stubtest).

I got 1 false positive from stubdefaulter though, the signature of `locateAll` depends on if openCV is installed. (it becomes `_locateAll_opencv` or `_locateAll_python`). In the stubs, we represent this through an overload, and stubdefaulter  gave the same defaults to both overloads.

**Comment by JelleZijlstra** [comment]
Thanks for the explanation! For `@overload` we currently just fill all defaults from the implementation (I reasoned about why at JelleZijlstra/stubdefaulter#22). That's been helping @AlexWaygood find a bunch of incorrect stubs in the stdlib, so I think on balance it's the right call. Your case here sounds very unusual.

**Comment by Avasam** [comment]
> [...] so I think on balance it's the right call. Your case here sounds very unusual.

I agree. It's a special case that required taking a special look at it. It was also obvious to spot and other tools caught it.

File: stubs/PyScreeze/pyscreeze/__init__.pyi [hunk 1]
```diff
@@ -16,11 +16,12 @@ _R = TypeVar("_R")
...
 _Mat: TypeAlias = Incomplete
 
-useOpenCV: bool
-RUNNING_PYTHON_2: Literal[False]
-GRAYSCALE_DEFAULT: Literal[False]
+useOpenCV: Final[bool]
+RUNNING_PYTHON_2: Final = False
+GRAYSCALE_DEFAULT: Final = False
+scrotExists: Final[bool]
+# Meant to be overrisable for backward-compatibility
```
Comment (JelleZijlstra) [inline stubs/PyScreeze/pyscreeze/__init__.pyi:23]:
```suggestion
# Meant to be overridable for backward-compatibility
```

**Review by JelleZijlstra — APPROVED** [review]

**Pull request merged**